collectors/0/old_flume_jmx.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+
+import sys
+from flume_jmx import FlumeJmxMonitor
+
+class OldFlumeJmxMonitor(FlumeJmxMonitor):
```

Comment:
yeah, I'm also not a fan of the name...but not really sure there's a better one out there.